alpine-openjdk/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM smebberson/alpine-base
```

Comment:
Would you mind fixing this to a particular release? I started this way when doing these images, but I quickly ran into issues so now the rule is, they must be fixed :)